build-go.sh

```diff
@@ -201,6 +201,14 @@ function checkoutVersion() {
 	test $? -eq 0 || fail "failed to check out $ref in $reporoot"
 }
 
+function installDeps() {
+	local repopath=$1
+
+	reporoot=$(cd "$repopath" && git rev-parse --show-toplevel)
+
+	(cd "$reporoot" && make deps)
```

Comment:
It might be over-engineering but calling:
```bash
(cd "$reporoot" && make -n deps >/dev/null 2>&1 && make deps)
```
seems much cleaner to me.
`-n` flag makes dry run so we can check if given rule exists.